optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -881,6 +881,64 @@ This avoids hard coding a solver time, because the unit test might run on arbitr
 This approach ensures that the test runs long enough to find a feasible solution, even on slow machines.
 But it does not run a millisecond longer than it strictly must, even on fast machines.
 
+Additionally, if you use a `ConstraintProvider` and have `org.optaplanner:optaplanner-test` as a dependency in your pom, you can test per constraint using a
+`ConstraintVerifier`.
+
+Add the `org.optaplanner:optaplanner-test` dependency to your `pom.xml` dependencies section:
+[source,xml]
+----
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+----
+
+Create the `src/test/java/com/example/solver/TimeTableConstraintProviderTest.java` class:
```

Comment:
Personally, I would reduce it to just the fields and the test method - the boilerplate does more bad than good, IMO. I think it is safe to assume that people know how to write a test.

Reply:
That go against the style of the rest of the guide; the guide shows the full code, even getters and setters (i.e. can be copied and pasted into an IDE and work).

Reply:
OK, I grudgingly accept.